### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jpmolina-bigid**

### CODE REVIEW

**Comment by mgraffia-bigid** [comment]
Do we need to commit the `bin` and `appserver/js/build` directories?  They seem like output.

**Comment by mgraffia-bigid** [comment]
> Do we need to commit the `bin` and `appserver/js/build` directories? They seem like output.

I might need the `bin` folder to be removed before I can view the rest of the files, my browser can't handle the amount of files with changes.

**Comment by jpmolina-bigid** [comment]
> > Do we need to commit the `bin` and `appserver/js/build` directories? They seem like output.
> 
> I might need the `bin` folder to be removed before I can view the rest of the files, my browser can't handle the amount of files with changes.

These two folders are generated by Splunk addon builder, the only file that I wrote is bin/input_module_bigid_audit_input.py

I tried deleting the appserver/js/build directory but the addon execution fails. My guess is that it is possible to delete some files but not all of them, but I haven't found documentation related to this yet

**Review by wissamaa — APPROVED** [review]

**Comment by wissamaa** [comment]
> Do we need to commit the `bin` and `appserver/js/build` directories? They seem like output.

yes, we need to commit these in order to be able to continue using the Add-on Builder. For example, checkout the Palo Alto Add-on for Splunk https://github.com/PaloAltoNetworks/Splunk_TA_paloalto

File: TA-bigid-audit-logs/app.manifest [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "schemaVersion": "2.0.0",
+  "info": {
+    "title": "BigID Audit logs",
+    "id": {
+      "group": null,
+      "name": "TA-bigid-audit-logs",
+      "version": "1.0.0"
```
[the remark this answers is not in the document]
Comment (jpmolina-bigid) [reply TA-bigid-audit-logs/app.manifest:8]: Should we add something else to the author details?

[the remark this answers is not in the document]
Comment (wissamaa) [reply TA-bigid-audit-logs/app.manifest]: author:
 -name: BigID Engineering
 -email: support@bigid.com
 -company: BigID

BigID Engineering is a catch all - inclusive of services, solutions, partner and product engineering

**Pull request merged**